packages/snap-preact-components/src/components/Molecules/Carousel/Carousel.tsx

```diff
@@ -243,8 +244,9 @@ export const Carousel = observer((properties: CarouselProps): JSX.Element => {
 			theme: props.theme,
 		},
 	};
+	const swiperModules = modules ? [Navigation, Pagination].concat(modules!) : [Navigation, Pagination];
```

Comment:
What would happen if someone passed in `[Navigation, Pagination]` ?
Maybe we should de-dupe somehow?